src/components/Atoms/Text/Text.js

```diff
@@ -31,6 +31,11 @@ export const BaseText = styled.span`
           }
         `
     : null)};
+    ${({ mobileColor, theme }) => mobileColor && css`
+    @media (max-width: 1023px) {
```

Comment:
Check out some other components to see how we use variables within media queries: https://github.com/comicrelief/component-library/blob/master/src/components/Molecules/Box/Box.js#L12

Which makes use of: https://github.com/comicrelief/component-library/blob/dcf690c82974595e65c1bdb918144cc9df5c1db4/src/theme/shared/breakpoint.js